### ISSUE TITLE
Feature request: Ability to select joystick inputs rather than record them in conditions and macros

### ISSUE THREAD

**Issue by TheGoodIdeaFairy** [WhiteMagic/JoystickGremlin#413]
Some physical buttons are difficult or impossible to isolate for the press-to-set method in the Joystick Condition edit mode.  For instance, buttons which fire when an axis is moved to a given point will always result in JG grabbing the axis instead of the button.

This would not be a problem if the interface for Joystick Conditions were changed to match the UI of vJoy Conditions.

Thank you, all.

**Comment by WhiteMagic** [comment]
I can see where this would be useful, however, those are very specific scenarios. For the large majority of the use cases, the current method is simpler and faster to use. Thus the default behavior will not be able to change as that would make things worse. So the trick is to find a way to have an alternative recording mode exposed without things getting a nightmare to use.

**Comment by TheGoodIdeaFairy** [comment]
Yes, luckily in my specific case Virpil has a built in workaround; they expose to the user an intermediate layer (logical button assignment) between the physical buttons and the input received by the computer, and this can be configured easily to move the otherwise inaccessible logical buttons to different physical inputs.  I don't know how common this degree of customizability is among the various joystick producers, though, so I'm unsure how universal the workaround is.
